extensions/youtube-subscriber-count/src/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { Icon, MenuBarExtra, open, LocalStorage } from "@raycast/api";
+import { useFetch } from "@raycast/utils";
+import { getPreferenceValues } from "@raycast/api";
+import { ActionPanel, Action, Detail, openExtensionPreferences } from "@raycast/api";
+
+type Item = {
+  statistics: {
+    subscriberCount: string;
+  };
+};
+
+type Data = {
+  items: Item[];
+};
+
+export default function Command() {
+  const preferences = getPreferenceValues();
```

Comment:
```suggestion
  const preferences = getPreferenceValues<Preferences>();
```

This will make a few errors in the linter, please also look into them